### PR TITLE
Silviya vasileva patch 1

### PR DESCRIPTION
**Pull request by SilviyaVasileva**
Adding new menu

### CODE REVIEW

File: menu-pages/schoolAid-forum.html [hunk 1]
```diff
@@ -107,6 +108,10 @@
             </li>
         </ul>
     </nav>
+    <span class="openbtn" onclick="openNav()">&#9776; Меню</span>
+
+
+
```
Comment (Veniyana) [inline menu-pages/schoolAid-forum.html]: a div wrapper so that the JS will work

File: menu-pages/schoolAid-login.html [hunk 2]
```diff
@@ -107,6 +108,10 @@
             </li>
         </ul>
     </nav>
+    <span class="openbtn" onclick="openNav()">&#9776; Меню</span>
+
+
+
```
Comment (Veniyana) [inline menu-pages/schoolAid-login.html]: same here..look at the menu-pages/schoolAid-course.html . I changed it to give you a guide

File: menu-pages/schoolAid-main.html [hunk 3]
```diff
@@ -108,6 +109,9 @@
             </li>
         </ul>
     </nav>
+    <span class="openbtn" onclick="openNav()">&#9776; Меню</span>
+
+
```
Comment (Veniyana) [inline menu-pages/schoolAid-main.html]: here as well

File: menu-pages/schoolAid-registration.html [hunk 4]
```diff
@@ -107,6 +108,10 @@
             </li>
         </ul>
     </nav>
+    <span class="openbtn" onclick="openNav()">&#9776; Меню</span>
+
+
+
```
Comment (Veniyana) [inline menu-pages/schoolAid-registration.html]: same

File: menu-pages/schoolAid-supply.html [hunk 5]
```diff
@@ -107,6 +108,10 @@
             </li>
         </ul>
     </nav>
+    <span class="openbtn" onclick="openNav()">&#9776; Меню</span>
+
+
+
```
Comment (Veniyana) [inline menu-pages/schoolAid-supply.html]: same

File: subject/biology-course.html [hunk 6]
```diff
@@ -107,6 +108,10 @@
             </li>
         </ul>
     </nav>
+    <span class="openbtn" onclick="openNav()">&#9776; Меню</span>
+
+
+
```
Comment (Veniyana) [inline subject/biology-course.html]: same?...why do we have so many <br>?

**Review by Veniyana — CHANGES_REQUESTED** [review]
please wrap the page content without the menu and nav in a <div class="content" id="content"> so that the js will work

File: subject/biology-course.html (hunk 6 above)
Comment (SilviyaVasileva) [reply subject/biology-course.html]: idk....

Comment (Veniyana) [reply subject/biology-course.html]: let's keep them later on we can delete them

**Review by Veniyana — APPROVED** [review]

**Pull request merged**